tests/unit/test_db_models.py

```diff
@@ -1,14 +1,15 @@
 """Tests for DB models."""
-from unittest import TestCase
+import pytest
 from pydantic import ValidationError
 
-from db.models import EVCBaseDoc, DocumentBaseModel, TAGDoc
+from db.models import EVCBaseDoc, DocumentBaseModel, TAGDoc, EVCUpdateDoc
 
 
-class TestDBModels(TestCase):
+class TestDBModels():
```

Comment:
Nicely done, taking the opportunity to also fix this tech debt.

Reply:
Ty